buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -674,6 +674,13 @@ class BeamModulePlugin implements Plugin<Project> {
         'varargs',
       ]
 
+      // Enable errorprone static analysis
+      project.apply plugin: 'net.ltgt.errorprone'
+
+      project.dependencies {
+        errorproneJavac 'com.google.errorprone:error_prone_core:2.3.1'
```

Comment:
It might make sense to upgrade errorprone_core and errorprone_annotations to 2.3.3

Reply:
Yea, at first I tried that and then I backed down when I hit errors. Since the errors do not seem related, might as well upgrade them also. It may make it more likely to work as the plugin expects to use the most recent errorprone prior to the plugin's release.